### ISSUE TITLE
Gerund features on present participles

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_English-EWT#506]
I found one example here:

```
# sent_id = answers-20111107214231AA68BMD_ans-0002
# newpar id = answers-20111107214231AA68BMD_ans-p0002
# text = I'm considering taking a job with Steiner and noticed I have to pay for all my travel.
1-2     I'm     _       _       _       _       _       _       _       _
1       I       I       PRON    PRP     Case=Nom|Number=Sing|Person=1|PronType=Prs      3       nsubj   3:nsubj|4:nsubj:xsubj|10:nsubj  _
2       'm      be      AUX     VBP     Mood=Ind|Number=Sing|Person=1|Tense=Pres|VerbForm=Fin   3       aux     3:aux   _
3       considering     consider        VERB    VBG     VerbForm=Ger    0       root    0:root  _
4       taking  take    VERB    VBG     VerbForm=Ger    3       xcomp   3:xcomp _
```

I think `considering` is a participle here.  Haven't looked for others yet

**Comment by nschneid** [comment]
I think this is #305

**Comment by nschneid** [comment]
Fixed in c6ed51e
